tests/test_model_utils.py

```diff
@@ -0,0 +1,59 @@
+from os.path import join, exists
+from pytest import mark, raises
+import torch
+from torchmdnet.models.utils import Distance
+
+
+@mark.parametrize("cutoff_lower", [0, 2])
+@mark.parametrize("cutoff_upper", [5, 10])
+@mark.parametrize("return_vecs", [False, True])
+@mark.parametrize("loop", [False, True])
+def test_make_splits_ratios(cutoff_lower, cutoff_upper, return_vecs, loop):
```

Comment:
The name of the test is strange.

Reply:
Good catch, copy paste mistake. I've changed it now.